proxy/contracts/schain/TokenManager.sol

```diff
@@ -188,5 +196,7 @@ abstract contract TokenManager is AccessControlUpgradeable, IMessageReceiver {
         tokenManagerLinker = newIMALinker;
         communityLocker = newCommunityLocker;        
         depositBox = newDepositBox;
+
+        emit DepositBoxWasChanged(address(0), newDepositBox);
```

Comment:
Maybe let's add different event?

Reply:
Do you think that this event does not describe what happen?

Reply:
We emit this event in initialize function